lms/templates/courseware/info.html

```diff
@@ -77,7 +77,11 @@ <h2 class="page-subtitle">${course.display_name | h}</h2>
       <h1>${_("Course Updates and News")}</h1>
       ${get_course_info_section(request, masquerade_user, course, 'updates')}
       % if self.course_info_extra_file:
-        <%include file="${self.course_info_extra_file}" />
+        % if sneakpeek_allowed and not regularly_registered:
+          <%include file="course_info_extra_sneakpeek.html" />
+        % else:
+          <%include file="${self.course_info_extra_file}" />
```

Comment:
Doh! I need to read more carefully, nevermind, I'm an idiot...both of these have the links to the ToS and PP.